command/helper/config.go

```diff
@@ -66,7 +66,7 @@ func DefaultConfig() *Config {
 		BlockGasTarget: "0x0", // Special value signaling the parent gas limit should be applied
 		Network: &Network{
 			NoDiscover: false,
-			MaxPeers:   20,
+			MaxPeers:   30,
```

Comment:
Increase to 50 as discussed

Reply:
Sorry, Missed it. Here is the commit [link](https://github.com/0xPolygon/polygon-sdk/pull/289/commits/cdecd53d6c7a1454f58469b80d15b124f0e4f506)